main_server/local_server/install_wizard/installer.js

```diff
@@ -464,6 +464,7 @@ function BMLTInstaller( in_prefs    ///< A JSON object with the initial prefs.
                     {
```

Comment:
It's good to get into the habit of using `===` in javascript. I'd change this to `=== true`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators